### ISSUE TITLE
BUG - Low - iOS - Accessibility:  Voiceover reads

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va-mobile-app#12185]
## What happened?

WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

Claim Details → What You Have Claimed section.
VoiceOver does not recognize the bullet as a list. If a list is visually presented but not coded as a proper list, it fails this criterion.

The bulleted list element(s) are not read as a list with Voiceover.

## Specs:

- Device: iPhone
- OS: iOS
- User Account: Demo
- Accessibility State: Voiceover ON

## Steps to Reproduce

1. Log into default demo user
2. Navigate to Claims > Claims history
3. Tap first Claim dated December 05, 2021
4. With Voiceover turned on, read the "What you've claimed" bulleted list (* Asthma, bronchial (increased), *hypertension (New))
## Desired behavior

Bulleted list should be noted that it is a list on Voiceover.
## Acceptance Criteria

Bulleted list should be noted that it is a list on Voiceover.
## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact:  Low 
- Frequency:  Low 

- 3 - Low

## Screen shot(s) and additional information

https://github.com/user-attachments/assets/fb654167-d80c-4a8a-904b-b66597be3e9d

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))
